dotnet/ITS.Propagation.P528/LineOfSightParams.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
```

Comment:
Since this file only defines a structure, it seems most of these are not used. If that is the case, you could remove the unused ones. This applies to the other files as well.

Reply:
Cleaned up